static/lookupConfig.json

```diff
@@ -823,6 +823,11 @@
     "uri": "https://id.loc.gov/vocabulary/carriers",
     "component": "list"
   },
+  {

```

Comment:
Duplicate needs removing.

Reply:
oops. I've made the change to sfolsom-patch-1. (I think I did it correctly, first time updating a patch.)